src/pip/_internal/exceptions.py

```diff
@@ -324,6 +325,7 @@ def __str__(self):
             message_part = " in {}.".format(self.fname)
         else:
             assert self.error is not None
-            # message attribute for Exception is only available for Python 2
+            # configparser.Error is missing "message" attribute in typeshed
+            # for python 3 (https://github.com/python/typeshed/issues/4058)
             message_part = ".\n{}\n".format(self.error.message)  # type: ignore
```

Comment:
This `message attribute for `configparser.Error` is not documented. That class as a proper support for `str()` which returns message, so

```suggestion
            message_part = ".\n{}\n".format(self.error)
```

Reply:
Thanks, that's a good idea, fixed in latest commit.